lib/forwarder.js

```diff
@@ -10,6 +10,47 @@ import { removeConfig } from '../lib/config.js';
 const EXIT_TOKEN_REGEXP = new RegExp(/EXIT([0-9]{3})/);
 const EXIT_TOKEN_LENGTH = 7;
 
+/**
+ * @param {Config | null} config
+ * @returns {Promise<boolean>}
+ */
+export function isAlive(config) {
+  return forwardCommandToDaemon(config, 'ESLINT_D_PING')
+    .then((response) => JSON.parse(response))
+    .then((result) => result.isAlive)
```

Comment:
I don't think we need a JSON response protocol for commands. The two implemented commands are "acknowledged" when sending the command is completing without any errors. I'd remove the response handling which will also simplify the `forwardCommandToDaemon` code.

Reply:
Sounds good to me, done